integration/js/checks/VideoBackgroundBlurCheck.js

```diff
@@ -2,33 +2,46 @@ const AppTestStep = require('../utils/AppTestStep');
 const { KiteTestError, Status, TestUtils } = require('kite-common');
 
 class VideoBackgroundBlurCheck extends AppTestStep {
-  constructor(kiteBaseTest, sessionInfo, attendeeId) {
+
+  filter_type;
+
+  constructor(kiteBaseTest, sessionInfo, attendeeId, filter_type) {
     super(kiteBaseTest, sessionInfo);
     this.attendeeId = attendeeId;
+    this.filter_type = filter_type;
   }
-  static async executeStep(KiteBaseTest, sessionInfo, attendeeId) {
-    const step = new VideoBackgroundBlurCheck(KiteBaseTest, sessionInfo, attendeeId);
+  static async executeStep(KiteBaseTest, sessionInfo, attendeeId, filter_type) {
+    const step = new VideoBackgroundBlurCheck(KiteBaseTest, sessionInfo, attendeeId, filter_type);
     await step.execute(KiteBaseTest);
   }
 
   stepDescription() {
-    return 'Check video blur';
+    return 'Check background filter';
```

Comment:
Can we do something like `Check background ${this.filtertype === 'replacment? 'replacement' : 'blur} `?